wicket-native-websocket/wicket-native-websocket-core/src/test/java/org/apache/wicket/protocol/ws/util/tester/WebSocketTesterProcessorTest.java

```diff
@@ -59,6 +59,11 @@ protected void onOutMessage(byte[] message, int offset, int length)
 		{
 			messageReceived.set(true);
 		}
+
+		@Override
+		protected void onOutMessage(byte[] message) {
+			messageReceived.set(true);
```

Comment:
I think this test should call only the old callback method. 
This method body should be empty or throw an Exception if it is called by mistake.